server/service/src/sync/translations/activity_log.rs

```diff
@@ -0,0 +1,210 @@
+use chrono::NaiveDateTime;
+use repository::{
+    ActivityLogRow, ActivityLogRowRepository, ActivityLogType, ChangelogRow, ChangelogTableName,
+    StorageConnection, SyncBufferRow,
+};
+use serde::{Deserialize, Serialize};
+
+use crate::sync::sync_serde::empty_str_as_option;
+
+use super::{
+    IntegrationRecords, LegacyTableName, PullUpsertRecord, PushUpsertRecord, SyncTranslation,
+};
+
+#[derive(Deserialize, Serialize, Debug, PartialEq)]
+pub enum LegacyActivityLogType {
+    #[serde(rename = "user_logged_in")]
```

Comment:
Not too sure what the types will look like on mSupply 🤔

Reply:
it's a new table on mSupply is it? Not mapping to the types in the `log` table?

Reply:
I mapped the type as is from omSupply

Reply:
> it's a new table on mSupply is it? Not mapping to the types in the log table?

Log table doesn't currently sync, should it be syncing ? (I think it's a lot of work to sync existing log table, so decided it should be new one)

Reply:
no - shouldn't be syncing. I remember deciding that we'd use a new one. Then got myself confused when reviewing, and had a look at the types in the existing mSupply table...